cadwyn/applications.py

```diff
@@ -85,7 +85,7 @@ def __init__(
         root_path_in_servers: bool = True,
         responses: dict[int | str, dict[str, Any]] | None = None,
         callbacks: list[BaseRoute] | None = None,
-        webhooks: APIRouter | None = None,
```

Comment:
It doesn't have to be a VersionedAPIRouter. Any APIRouter will do. The only difference between them is that VersionedAPIRouter stores info about deleted routes, which we do not care about here because we do not really support marking webhooks as **deleted** yet.